module/bdev/aio/bdev_aio.c

```diff
@@ -231,13 +320,134 @@ bdev_aio_writev(struct file_disk *fdisk, struct spdk_io_channel *ch,
 	return len;
 }
 
+static struct aio_request_ctx *
+create_aio_request_ctx(struct spdk_bdev_io *bdev_io, aio_request_fn fn, size_t ctx_size)
+{
+	struct aio_request_ctx *request = malloc(sizeof(struct aio_request_ctx) + ctx_size);
```

Comment:
malloc memory is typically garbage, calloc sets it to zero IIRC. in general, everywhere in SPDK calloc is used so let us also use that?

Also, instead of doing sizeof(struct ..)) sizeof( *aio_request_ctx) can be used (saves typing when redoing code) and is more consistent with the rest of the code.